data/sponsors.json

```diff
@@ -0,0 +1,33 @@
+{
+    "sponsors":{
+        "Platinum":[
+            {
+                "alt-text":"microsoft",
+                "img":"img/micro.png"
+            },
+            {
+                "alt-text":"microsoft",
+                "img":"img/micro.png"
+            },
+            {
+                "alt-text":"microsoft",
+                "img":"img/micro.png"
+            }
+        ],
+        "Gold":[
+            {
+                "alt-text":"microsoft",
+                "img":"img/micro.png"
+            },
+            {
+                "alt-text":"microsoft",
+                "img":"img/micro.png"
+            },
+            {
+                "alt-text":"microsoft",
+                "img":"img/micro.png"
+            }
+        ]
+    },  
+    "border":"img/border2.png"
```

Comment:
You can just remove the border key and hard code it in the image, since this value is  going to be fixed.